package.json

```diff
@@ -1622,12 +1610,14 @@
                 "jupyter.allowLiveShare": {
                     "type": "boolean",
                     "default": true,
+                    "deprecationMessage": "This setting is deprecated as Live Share support for the interactive window no longer exists in the Jupyter extension.",
```

Comment:
We should probably enter a task on us to reimplement this. The liveshare team has a bug where people are complaining about this being removed: https://github.com/MicrosoftDocs/live-share/issues/4032

Reply:
I opened https://github.com/microsoft/vscode-jupyter/issues/7460